home/dot_config/shell/exports.sh.tmpl

```diff
@@ -48,6 +48,7 @@ export PATH="$HOME/.local/bin/cask:$PATH"
 export PATH="$HOME/.local/bin/docker:$PATH"
 export PATH="$HOME/.local/bin/firejail:$PATH"
 export PATH="$HOME/.local/bin/flatpak:$PATH"
+export PATH="$HOME/.local/bin/mackup:$PATH"
```

Comment:
I don't think this is required. All these imports are actually importing folders (i.e. `.local/bin/firejail` will contain Firejail wrappers for programs). You can remove this line.

Reply:
The wrapper script in this repository is used to link to XDG directories before running the actual `mackup` and then unlink them. If the wrapper is to be saved under `~/.local/bin`, pip based installation will overwrite. So, the wrapper is placed in a separate folder and it is added to PATH before `~/.loca/bin`. If this `export` is removed, wrapper will not be executed and `mackup` will fail as it will not see the config created by this repository (no XDG support yet). This has been tested. Additional info is available [here](https://github.com/megabyte-labs/install.doctor/pull/42#issuecomment-1600727490).